src/NuGet.Core/NuGet.Protocol/LegacyFeed/V2FeedQueryBuilder.cs

```diff
@@ -142,14 +142,15 @@ public string BuildGetPackageUri(PackageIdentity package)
             // Start building the URI.
             var builder = new StringBuilder();
 
-            builder.AppendFormat(GetPackagesFormat, useParenthesis ? EndpointParenthesis : string.Empty);
+            builder.AppendFormat(CultureInfo.CurrentCulture, GetPackagesFormat, useParenthesis ? EndpointParenthesis : string.Empty);
```

Comment:
Since this involves an URI, it may be good to use an invariant culture. This comment applies to all changes in this file.

```suggestion
            builder.AppendFormat(CultureInfo.InvariantCulture, GetPackagesFormat, useParenthesis ? EndpointParenthesis : string.Empty);
```